lib/Backend/NativeCodeGenerator.cpp

```diff
@@ -2014,10 +2008,6 @@ NativeCodeGenerator::UpdateQueueForDebugMode()
 
     this->hasUpdatedQForDebugMode = true;
 
-    if (Js::Configuration::Global.EnableJitInDebugMode())
-    {
-        Processor()->AddManager(this);
-    }
```

Comment:
As above - need to keep `Processor()->AddManager(this);`, just without the `if`.